pkg/download/jsoncreator/json_creator.go

```diff
@@ -112,11 +112,12 @@ func processJSONFile(data map[string]interface{}, id string) ([]byte, error) {
 
 //replaceKeyProperties replaces name or displayname for each config
 func replaceKeyProperties(dat map[string]interface{}) map[string]interface{} {
-	//removes id field
-	delete(dat, "id")
 
-	// Removes metadata field
-	delete(dat, "metadata")
+	dat = removeKey(dat, []string{"metadata"}) // Removes metadata field
```

Comment:
nitpick: you could remove the comments, it's somewhat obvious what this line does